src/app/settings/containers/settings/settings.component.html

```diff
@@ -14,7 +14,7 @@
   ></cs-api-settings>
   <cs-vm-settings
     [settings]="settings$ | async"
-    (updateKeyboardLayout)="keyboardChange($event)"
+    (updateKeyboardLayout)="onUpdateKeyboardLayout($event)"
```

Comment:
Make it looks like other bindings in the components.
It looks that its logic is different